vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/Reasoner.java

```diff
@@ -86,6 +87,22 @@ static Reasoner getInstance() {
 	 */
 	KnowledgeBase getKnowledgeBase();
 
+	/**
+	 * Exports all the facts of the knowledge base to an OutputStream. This includes
+	 * the added facts and the inferred facts as well.
+	 * 
+	 * @param an OutpumStream for the facts to be written to.
+	 */
+	void getKbFacts(OutputStream stream) throws IOException;
```

Comment:
usually methods named "get" return something, maybe rename to "writeKBFacts", or even better, "writeReasoningResult"?